tests/components/cloud/test_google_config.py

```diff
@@ -341,12 +341,13 @@ async def test_sync_google_on_home_assistant_start(
     config = CloudGoogleConfig(
         hass, GACTIONS_SCHEMA({}), "mock-user-id", cloud_prefs, hass.data["cloud"]
     )
-    hass.set_state(CoreState.starting)
+    hass.set_state(CoreState.not_running)
```

Comment:
Do we need to update also other cloud tests which start by setting the core state to starting?

Reply:
Yes there are some other ones that need to be adjusted as well

Reply:
Followup https://github.com/home-assistant/core/pull/113045